### DISCUSSION TITLE
Feature Request: Ability to pause dashboard playlists

### DISCUSSION THREAD

**Discussion by nirajsanghvi** [grafana/grafana#66426] [Dashboards, Panels & Visualizations]
When running a playlist, there's controls shown at the top to manually switch to the previous/next dashboard and to stop the playlist. It would be quite useful to also have a pause option since pressing stop causes the controls to go away completely, and you may have just temporarily wanted to inspect a dashboard or some metric more closely (for longer than the usual change interval).

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82717. Feel free to continue the discussion around this there. Thank you!
